classes/01class/exercises/c01-aws04-deprecated/README.md

```diff
@@ -1,4 +1,4 @@
-# Tools in use (C01-E01)
+# Tools in use (c01-aws04 - deprecated)
```

Comment:
Should remove this from tracked lab exercises in `/scripts/labs.txt`?

Reply:
I'm removing that in the next PR.